exercises/practice/yacht/.meta/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "authors": [
+    "glennj"
+  ],
+  "contributors": [
+    "m-dango"
+  ],
+  "files": {
+    "solution": [
+      "lib/Yacht.pm"
+    ],
+    "test": [
+      "t/yacht.t"
+    ],
+    "example": [
+      ".meta/solutions/lib/Yacht.pm"
+    ],
+    "editor": [
+      "lib/Yacht/Category.pm"
+    ]
```

Comment:
Note this. This makes the Category module visible in the web editor, but readonly.